js/customizer.js

```diff
@@ -11,32 +11,26 @@
 	// Site title and description.
 	wp.customize( 'blogname', function( value ) {
 		value.bind( function( to ) {
-			$( '.site-title a' ).text( to );
-		} );
-	} );
+			document.querySelector( '.site-title a' ).textContent( to );
```

Comment:
I think this is invalid since `textContent` is not a function.
```suggestion
			document.querySelector( '.site-title a' ).textContent = to;
```